### PR TITLE
Language changes

### PR DESCRIPTION
**Pull request by ath0111**
1. Added Date-Requested in Request fields.
2. Changed button name in Request Detail : Shipping Info => Get Shipping Label
3. Made logo link to "http://collectiveshield.org"

Screen Shot 2020-04-10 at 3 11 00 PM
Screen Shot 2020-04-10 at 3 10 45 PM

### CODE REVIEW

File: ui/src/components/Navbar.tsx [hunk 1]
```diff
@@ -16,7 +16,11 @@ const MainNav: React.FC<{ user: User | undefined }> = ({ user }) => {
       <Container>
         <Row className="justify-content-end nav-inner no-gutters">
           <Col xs={12} lg={3}>
-            <Navbar.Brand href="/" className="branding">
+            <Navbar.Brand
+              href="http://collectiveshield.org"
```
Comment (mjoyce91) [inline ui/src/components/Navbar.tsx:20]: can we do `https://www.collectiveshield.org`?

Comment (ath0111) [reply ui/src/components/Navbar.tsx:20]: Yup!

File: ui/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "input-moment": "^0.4.0",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
-    "node-sass": "4.13.0",
+    "node-sass": "^4.13.1",
```
Comment (mjoyce91) [inline ui/package.json:18]: Did this need to be updated?

Comment (ath0111) [reply ui/package.json:18]: It was upgraded when I did `npm install`

**Pull request merged**